src/hooks/helpers/useLoader.test.ts

```diff
@@ -0,0 +1,117 @@
+import { renderHook, act } from '@testing-library/react';
+import { useLoader } from './useLoader';
+import { STATUS } from '../../types';
+
+describe('useLoader', () => {
+	test('Init', () => {
+		const defaultStatus = STATUS.REQUESTING;
+		const defaultDataState = {};
+		const { result } = renderHook(() => useLoader(defaultStatus, defaultDataState));
+
+		expect(result.current.data).toEqual(defaultDataState);
+		expect(result.current.loadingStatus.status).toEqual(defaultStatus);
+	});
+
+	test('setLoading', () => {
```

Comment:
I feel like this test name should be a little more descriptive. Based on the code here, it looks like what it's actually testing is that _after_ we call `setSuccess`, does calling `setLoading` preserve the `data` while changing the status to `REQUESTING`

Maybe the test name should resemble something to that effect